pages/tips.md

```diff
@@ -0,0 +1,55 @@
+
+The answer is probably "yes," but nobody can help you until you state what your problem is. There are several questions to this effect every week, and all too often by the time somebody asks what the real question is, the person who asked has disconnected.
+
+### "Is it okay if I ask about [X tool or library] here?"
+
+We have a dizzying array of channels, which we're aware of and for which we apologize. We have 10 or 15 various utility channels grouped at the top (announcements, random, q-and-a for scheduled guests, etc), and the rest are grouped alphabetically. Instead of asking, scan through the list for any that seem like obvious fits for your question.
+
+If you don't see any, don't hesitate to post it in **#general** or **#need-help**. If there's a better place for it, one of the moderators or other helpful users will suggest it, but off topic messages aren't going to get you banned.
+
+## Don't ping or DM other devs you aren't actively talking to
+
+It's very tempting to try to get more attention to your question by @-mentioning one of the high profile (or recently active) members of Reactiflux, but please don't. They may not actually be online, they may not be able to help, and they may be in a completely different timezone--nobody likes push notifications at 3am from an impatient stranger.
+
+Similarly, don't DM other members without asking first. All of the same problems as @-mentioning apply, and private conversations can't help anyone else. Your questions are likely not unique, and other people can learn from them when they're kept public.
+
+## Be helpful
+
+It doesn't matter if you can't answer a question in a brief sentence, or if you get pulled away before you can type out further replies. If a question isn't clear, ask a clarifying question. If you remember something you read that seemed related, send a link. A lot of people who are stuck will be fine if you can't provide the solution, they just need another place to look.
+
+## Be polite
+
+The other people in Reactiflux deserve your respect, so be polite. Not "retail employee" polite, but don't fan flamewars, get sucked into lose-lose arguments, or other internet classics.
+
+## Be a model channel citizen
+
+Be the change you want to see in the world! If a conversation is getting heated, try to defuse it. If a hard question is unanswered, dig in and work through it with them. If a channel is full of unanswered questions, mention in **#reactiflux-dev** that you'd like to take ownership of it.
+
+---
+
+These are some general pointers; they're not comprehensive, and they're not a guarantee that you'll get answers to all your questions. We also have an _MVP_ designation for a small number of contributors. There aren't set conditions you need to meet to be considered for it, the people we privately consider for it have embodied these qualities for significant amount of time. Their presence in the channel makes it a better place for everyone.
```

Comment:
I moved the "MVP" mention to the bottom of the list, but I don't want to give it its own header because I don't think it's something that most users should focus on. I definitely want to mention it, but at a lower level of visibility than the tips.